src/components/loader-bar/loader-bar.test.tsx

```diff
@@ -3,51 +3,57 @@ import { render, screen } from "@testing-library/react";
 import LoaderBar from "./loader-bar.component";
 import useMediaQuery from "../../hooks/useMediaQuery";
 
-jest.mock("../../hooks/useMediaQuery", () => {
-  return {
-    __esModule: true,
-    default: jest.fn().mockReturnValue(true),
-  };
+jest.mock("../../hooks/useMediaQuery", () => ({
+  __esModule: true,
+  default: jest.fn(),
+}));
+
+const mockUseMediaQuery = useMediaQuery as jest.MockedFunction<
+  typeof useMediaQuery
+>;
+
+beforeEach(() => {
+  jest.clearAllMocks();
+  mockUseMediaQuery.mockReturnValue(false);
+});
+
+afterAll(() => {
+  jest.restoreAllMocks();
 });
 
 test("renders", () => {
   render(<LoaderBar />);
-
   const loaderBar = screen.getByRole("progressbar", { name: "Loading" });
   expect(loaderBar).toBeVisible();
 });
 
 test("renders with custom data role and element", () => {
   render(<LoaderBar data-role="foo" data-component="bar" data-element="baz" />);
-
   const loaderBar = screen.getByTestId("foo");
   expect(loaderBar).toHaveAttribute("data-role", "foo");
   expect(loaderBar).toHaveAttribute("data-element", "baz");
 });
 
 test("renders alternative loading text if user prefers reduced motion", () => {
-  const mockUseMediaQuery = useMediaQuery as jest.MockedFunction<
-    typeof useMediaQuery
-  >;
   mockUseMediaQuery.mockReturnValueOnce(false);
-
   render(<LoaderBar />);
-
   const loaderBar = screen.getByRole("progressbar", { name: "Loading" });
   expect(loaderBar).toHaveTextContent("Loading");
 });
 
-/* Styling test for coverage */
 it.each([
```

Comment:
comment: this should be `test` instead of `it` as there is no parent `describe` etc

Reply:
I'm not sure it needs to be async either as you are mocking the value of the `useMediaQuery` hook